packages/php-wasm/node/src/test/php.spec.ts

```diff
@@ -654,6 +654,31 @@ bar1
 			expect(out.errors).toBe('');
 			expect(messageReceived).toBe('world');
 		});
+
+		if (parseFloat(phpVersion) >= 8) {
```

Comment:
Just to avoid unreliable float comparisons:

```suggestion
		if (parseInt(phpVersion.substring(0,1)) >= 8) {
```